src/pages/blog/video-banner/_DBanner.js

```diff
@@ -0,0 +1,172 @@
+import React, { useState } from 'react'
+import styled from 'styled-components'
+import PropTypes from 'prop-types'
+import VideoSrc from '../../markets/static/video/globe.mp4'
+import VideoCarousel from './_VideoCarousel'
+import VideoPlayer from './_video-player'
+import { Flex, Container } from 'components/containers'
+import { Header, LinkText } from 'components/elements'
+import device from 'themes/device'
+import PlayIcon from 'images/svg/blog/video/Triangle.svg'
+
+const ParentWrapper = styled(Flex)`
+    /* prettier-ignore */
+    background: ${(props) =>
+        props.bg_image
+            ? 'linear-gradient(251.14deg,rgba(14, 14, 14, 0.5632) 29.18%,rgba(7, 6, 6, 0.88) 85.14%),url(' +
+              props.bg_image +
+              ') no-repeat top left'
+            : 'linear-gradient(251.14deg, rgba(14, 14, 14, 0.5632) 29.18%, rgba(7, 6, 6, 0.88) 85.14%)'};
+    flex-direction: column;
+    overflow: hidden;
+    margin: 80px 0;
+    position: relative;
+    background-size: cover;
+    height: auto;
+    padding: 160px 0 80px;
+    max-width: 1600px;
+
+    @media ${device.tabletL} {
+        /* prettier-ignore */
+        background: ${(props) =>
+            props.bg_image
+                ? 'linear-gradient(251.14deg, rgba(14, 14, 14, 0.5632) 29.18%, rgba(7, 6, 6, 0.88) 85.14%),url(' +
+                  props.bg_image +
+                  ') no-repeat top right 46.5%'
+                : 'linear-gradient(251.14deg, rgba(14, 14, 14, 0.5632) 29.18%, rgba(7, 6, 6, 0.88) 85.14%)'};
+        background-size: cover;
+        padding: 73px 0 40px;
+    }
+`
+
+const BgWrapper = styled(Container)`
+    flex-direction: column;
+    justify-content: flex-start;
+`
+const VideoDetailsWrapper = styled(Flex)`
+    height: auto;
+    flex-direction: column;
+    justify-content: flex-start;
+`
+const PlayerIconWrapper = styled(Flex)`
+    width: 64px;
+    height: 64px;
+    border-radius: 50%;
+    background: rgba(255, 255, 255, 0.32);
+    margin-bottom: 24px;
+    align-items: center;
+    cursor: pointer;
+    position: relative;
+`
+const PlayerIcon = styled.img`
+    width: ${(props) => props.width};
+    height: ${(props) => props.height};
+    position: absolute;
+    top: 50%;
+    left: 50%;
+    transform: translate(-40%, -50%);
+`
+const TagParentWrapper = styled(Flex)`
+    height: 22px;
+    justify-content: flex-start;
+    @media ${device.tabletL} {
+        margin-bottom: 8px;
+    }
+`
+const TagWrapper = styled(Flex)`
+    width: auto;
+    padding: 1px 8px;
+    background: var(--color-orange-3);
+    border-radius: 8px;
+    align-items: center;
+`
+const Tag = styled(LinkText)`
```

Comment:
I've actually just used a regular header/text for this as there isn't gonna be a tags page in phase 1.

Reply:
fixed in latest commit